8.platemap/scripts/make_platemap.r

```diff
@@ -0,0 +1,61 @@
+library(ggplot2)
+library(platetools)
```

Comment:
Recommend adding SupressWarnings here to avoid any unneeded outputs in notebook.

Reply:
appplied